articles/resources/relnotes.md

```diff
@@ -16,6 +16,23 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.12.xxxxxxx
+
+*Release date: July 1st, 2020*
+
+This release contains the following:
+
+- New `qdk-chem` tool for converting legacy electronic structure problem serialization formats (e.g.: FCIDUMP) to [Broombridge](xref:todo)
+- New functions and operations in the [`Microsoft.Quantum.Synthesis` namespace](xref:microsoft.quantum.synthesis) for coherently applying classical oracles using transformation- and decomposition-based synthesis algorithms.
```

Comment:
The link to `microsoft.quantum.synthesis` is broken. This may be since the auto generated files have not been merged yet. The M.Q.Synthesis namespace does not show up in navigation of the preview of this PR.

Reply:
That link will be broken until the docs are uploaded with this release, but you can check that the UID is correct from the beta builds of the docs (e.g.: https://github.com/MicrosoftDocs/quantum-docs-pr/blob/beta/v0.12.2006.2435/api/qsharp/Microsoft.Quantum.Synthesis.yml from two days ago).